services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/FlowCrudSpec.groovy

```diff
@@ -412,6 +416,7 @@ class FlowCrudSpec extends BaseSpecification {
         ]
     }
 
+    @Tags(SMOKE)
```

Comment:
I don't think this is a critical test case. Too rare to ever actually happen, potential consequences are not critical

Reply:
removed